php-checks/src/main/java/org/sonar/php/checks/ClassCouplingCheck.java

```diff
@@ -81,7 +85,6 @@
 
   @Override
   public void visitScript(ScriptTree tree) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1185)